tests/js/spec/views/settings/components/settingsSearch/index.spec.jsx

```diff
@@ -152,5 +152,6 @@ describe('SettingsSearch', function() {
       .simulate('click');
 
     expect(navigateTo).toHaveBeenCalledWith('/billy-org/', expect.anything());
+    navigateTo.mockClear();
```

Comment:
Are your tests running directly after this test?

I wonder if moving Line 9-11 into the `describe(...)` block would encourage the garbage collector to clear the mock? Otherwise, it'll be a good idea to tear down all 3 mocks in a `afterAll(...)

Reply:
No -- I copy/pasted the mock setup and expectation from here and below (the only places in the codebase `navigateTo` is mocked. In both places, it seems like the mock is only called once. But when I was running multiple tests with this mock (in `tests/js/spec/components/onboardingWizard/tasks.spec.jsx`) back-to-back, I got some false positives until I added the `.mockClear()` call. Figured it'd be good to add mock cleanup to the other places the mock is being used.

For some reason, the tests fail when the mock is declared inside the describe block or in a beforeEach statement. :/